bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java

```diff
@@ -348,13 +355,29 @@ public void createTable(HTableDescriptor desc, byte[] startKey, byte[] endKey, i
   /** {@inheritDoc} */
   @Override
   public void createTable(HTableDescriptor desc, byte[][] splitKeys) throws IOException {
-    createTable(desc.getTableName(), TableAdapter.adapt(desc, splitKeys));
+    CreateTableRequest createReq = CreateTableRequest.of(desc.getTableName().getNameAsString());
```

Comment:
Why isn't this done in `TableAdapter`?

Reply:
Thanks for the feedback. Previously I thought of migrating  `TableAdapter` with `v2.models.CreateTableRequest` later, now that is also updated.